src/runtime/entitlements-manager.js

```diff
@@ -192,7 +196,11 @@ export class EntitlementsManager {
    * @return {!Entitlements}
    */
   parseEntitlements(json) {
-    const isReadyToPay = json['isReadyToPay'];
+    const isReadyToPay = /** @type {boolean} */ (json['isReadyToPay']);
+    if (json['isReadyToPay']) {
```

Comment:
`if (isReadyToPay) {}`. But probably need to always set? Otherwise if RTP goes from true to false, you will never overwrite false? Or we could test that its `!= null`?

Reply:
fixed